### PR TITLE
openvpn: 2.4.3

### PR DESCRIPTION
**Pull request by leonklingele**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

### CODE REVIEW

**Comment by bfontaine** [comment]
```
Error: 1 problem in 1 formula
openvpn:
  * 'revision 1' should be removed
```

File: Formula/openvpn.rb [hunk 1]
```diff
@@ -1,10 +1,9 @@
 class Openvpn < Formula
   desc "SSL/TLS VPN implementing OSI layer 2 or 3 secure network extension"
   homepage "https://openvpn.net/index.php/download/community-downloads.html"
-  url "https://swupdate.openvpn.org/community/releases/openvpn-2.4.2.tar.xz"
-  mirror "https://build.openvpn.net/downloads/releases/openvpn-2.4.2.tar.xz"
-  sha256 "df5c4f384b7df6b08a2f6fa8a84b9fd382baf59c2cef1836f82e2a7f62f1bff9"
-  revision 1
+  url "https://swupdate.openvpn.org/community/releases/openvpn-2.4.3.tar.xz"
+  mirror "https://build.openvpn.net/downloads/releases/openvpn-2.4.3.tar.xz"
+  sha256 "7aa86167a5b8923e54e8795b814ed77288c793671f59fd830d9ab76d4b480571"
```
Comment (bfontaine) [inline Formula/openvpn.rb]: Both the CI and I get a different checksum here:
```
curl -sL 'https://swupdate.openvpn.org/community/releases/openvpn-2.4.3.tar.xz'|shasum -a 256
15e15fc97f189b52aee7c90ec8355aa77469c773125110b4c2f089abecde36fb  -
```

**Comment by ilovezfs** [comment]
Thanks for the PR @leonklingele!

@bfontaine the tarball's gpg signature was good so I think we're fine.

**Comment by leonklingele** [comment]
For anyone interested, here's the diff between the revision of v2.4.3 which I initially downloaded and the one which is available now: https://gist.github.com/leonklingele/2179552e891acec458a9bf786b5076db

- Revision 1 ("my version") includes files generated with automake 1.14.1
- Revision 2 (available now) includes files generated with automake 1.15

These seem to be the only differences.

**Pull request closed without merging**